src/app/custom/translations.tsv

```diff
@@ -1009,16 +1002,34 @@
+"ws_post-v1-sciencemetrixClass_title"  "Classification into scientific fields Science-Metrix"	"Classification en domaines scientifiques Science-Metrix"
+"ws_post-v1-sciencemetrixClass_description"  "Classes documents in English in the three levels of the Science-Metrix classification"  "Classe des documents en anglais dans les trois niveaux de la classification Science-Metrix"
+"ws_post-v1-textExtract_title"  "Transforming PDF into text"  "Transformation d'un PDF en texte"
+"ws_post-v1-textExtract_description"  "Transforms a PDF into text by excluding elements that would disrupt subsequent text mining processing"  "Transforme un PDF en texte en excluant les éléments qui perturberaient un traitement de fouille de texte ultérieur"
+"ws_post-v1-quantityExtract_title"  "Extraction of quantities"  "Extraction de quantités"
+"ws_post-v1-quantityExtract_description"  "Extracts quantities (e.g.: 5 kg, 6 weeks…) in a text in English"  "Extrait des quantités (ex: 5 kg, 6 weeks…) dans un texte en anglais"
+"ws_post-v1-bibCheck_title"  "Checking a bibliographic reference"  "Contrôle d'une référence bibliographic"
+"ws_post-v1-bibCheck_description"  "Checks a given bibliographic reference, checking its presence in Crossref while ensuring that the associated article is not retracted."  "Contrôle une référence bibliographique donnée, en vérifiant sa présence dans Crossref tout en s’assurant que l’article associé n’est pas rétracté"
+"ws_post-v1-countryDetect_title"  "Detecting the country of an affiliation"  "Détection du pays d’une affiliation"
+"ws_post-v1-countryDetect_description"  "Detects the country of origin of an affiliation-address, whether present or absent in it"  "Détecte le pays d’origine d’une affiliation-adresse, qu’il soit présent ou absent dans celle-ci"
+"ws_post-v1-funderTag_title"  "Extracting funders in an article"  "Extraction de financeurs dans un article"
+"ws_post-v1-funderTag_description"  "Detects funders in an English article"  "Détecte des financeurs dans un article en anglais"
+"ws_post-v1-chemTag_title"  "Extracting named entities in chemistry"  "Extraction d’entités nommées en chimie"
+"ws_post-v1-chemTag_description"  "Detects entities named in chemistry in an English text."  "Détecte, dans un texte en anglais, les entités nommées en chimie"
+"ws_post-v1-diseaseTag_title"  "Extracting disease named entities"  "Extraction d’entités nommées de maladies"
+"ws_post-v1-diseaseTag_description"  "Detects disease named entities in English text"  "Détecte des entités nommées de maladies sur des textes en anglais"
+"ws_post-v1-astroTag_title"  "Extracting named entities in astronomy"  "Extraction d’entités nommées en astronomie"
+"ws_post-v1-astroTag_description"  "Detects entities named in astronomy on English texts and distributes them between 16 predefined classes"  "Détecte des entités nommées en astronomie sur des textes en anglais et les répartit entre 16 classes prédéfinies"
 "r-filter_title"	"filter"	"filter"
```

Comment:
<!-- Issues highlight specific problems with the subject under review. These problems can be user-facing or behind the scenes. It is strongly recommended to pair this comment with a suggestion. If you are not sure if a problem exists or not, consider leaving a question.  -->
**issue:** ‏Tabulations instead of spaces
```suggestion
"ws_post-v1-entityTag_title"	"Extraction of different named entities"	"Extraction de différentes entités nommées"
"ws_post-v1-entityTag_description"	"Detects different named entities in texts (at least when the corpus includes texts in several languages: person, location, organization, other)"	"Détecte dans les textes, différentes entités nommées (a minima quand le corpus comporte des textes dans plusieurs langues : personne, localisation, organisme, autre)"
"ws_post-v1-IdRorDetect_title"	"Assigning ROR identifier"	"Attribution d'un identifiant ROR"
"ws_post-v1-IdRorDetect_description"	"Returns an ROR identifier when querying the API"	"Renvoie un identifiant ROR en interrogeant l'API"
"ws_post-v1-sciencemetrixClass_title"	"Classification into scientific fields Science-Metrix"	"Classification en domaines scientifiques Science-Metrix"
"ws_post-v1-sciencemetrixClass_description"	"Classes documents in English in the three levels of the Science-Metrix classification"	"Classe des documents en anglais dans les trois niveaux de la classification Science-Metrix"
"ws_post-v1-textExtract_title"	"Transforming PDF into text"	"Transformation d'un PDF en texte"
"ws_post-v1-textExtract_description"	"Transforms a PDF into text by excluding elements that would disrupt subsequent text mining processing"	"Transforme un PDF en texte en excluant les éléments qui perturberaient un traitement de fouille de texte ultérieur"
"ws_post-v1-quantityExtract_title"	"Extraction of quantities"	"Extraction de quantités"
"ws_post-v1-quantityExtract_description"	"Extracts quantities (e.g.: 5 kg, 6 weeks…) in a text in English"	"Extrait des quantités (ex: 5 kg, 6 weeks…) dans un texte en anglais"
"ws_post-v1-bibCheck_title"	"Checking a bibliographic reference"	"Contrôle d'une référence bibliographic"
"ws_post-v1-bibCheck_description"	"Checks a given bibliographic reference, checking its presence in Crossref while ensuring that the associated article is not retracted."	"Contrôle une référence bibliographique donnée, en vérifiant sa présence dans Crossref tout en s’assurant que l’article associé n’est pas rétracté"
"ws_post-v1-countryDetect_title"	"Detecting the country of an affiliation"	"Détection du pays d’une affiliation"
"ws_post-v1-countryDetect_description"	"Detects the country of origin of an affiliation-address, whether present or absent in it"	"Détecte le pays d’origine d’une affiliation-adresse, qu’il soit présent ou absent dans celle-ci"
"ws_post-v1-funderTag_title"	"Extracting funders in an article"	"Extraction de financeurs dans un article"
"ws_post-v1-funderTag_description"	"Detects funders in an English article"	"Détecte des financeurs dans un article en anglais"
"ws_post-v1-chemTag_title"	"Extracting named entities in chemistry"	"Extraction d’entités nommées en chimie"
"ws_post-v1-chemTag_description"	"Detects entities named in chemistry in an English text."	"Détecte, dans un texte en anglais, les entités nommées en chimie"
"ws_post-v1-diseaseTag_title"	"Extracting disease named entities"	"Extraction d’entités nommées de maladies"
"ws_post-v1-diseaseTag_description"	"Detects disease named entities in English text"	"Détecte des entités nommées de maladies sur des textes en anglais"
"ws_post-v1-astroTag_title"	"Extracting named entities in astronomy"	"Extraction d’entités nommées en astronomie"
"ws_post-v1-astroTag_description"	"Detects entities named in astronomy on English texts and distributes them between 16 predefined classes"	"Détecte des entités nommées en astronomie sur des textes en anglais et les répartit entre 16 classes prédéfinies"
"r-filter_title"	"filter"	"filter"
```

Reply:
I assume you used the `Resolve converstation` button, but you should have used `Commit suggestion`.
![image](https://github.com/user-attachments/assets/782eccac-52f7-44a4-8347-eef3bc777b9b)

`Resolve conversation` is here for more complex cases, where the commits are not suggested.